worker.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+"""Worker script for OpenDroneMap transformer
+"""
+import os
+import yaml
+
+# Override pylint to allow our trickery in setting up ODM
```

Comment:
what is 'our trickery' ... please clarify...

Reply:
Done